internal/configurationmanagers/kubernetes/crd/crd_injector.go

```diff
@@ -6,43 +6,140 @@ import (
 	"log"
 	"strings"
 
-	apiextensionsv1beta1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
+	apiextensionsv1 "k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1"
 	apiextensionsclientset "k8s.io/apiextensions-apiserver/pkg/client/clientset/clientset"
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	meta_v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-func createCRD(apiExtClient *apiextensionsclientset.Clientset) error {
-	secretlessCRD := &apiextensionsv1beta1.CustomResourceDefinition{
-		ObjectMeta: meta_v1.ObjectMeta{
-			Name: CRDFQDNName,
+// This schema should be updated to match secretless-resource-definition.yaml
```

Comment:
I think this comment is saying that the schema "should be kept up-to-date" to match the yada, yada, yada...?
When I first read this, I interpreted "should be updated" to mean it's a TODO that we still need to get to.